extensions/kubernetes/deployment/src/main/java/io/quarkus/kubernetes/deployment/KubernetesProcessor.java

```diff
@@ -32,6 +44,9 @@
 
 class KubernetesProcessor {
 
+    private static final String PROPERTY_PREFIX = "dekorate.";
+    private static final String ALLOWED_GENERATOR = "kubernetes";
```

Comment:
Would it be hard to have a test for this feature?

Reply:
It would have to be a `mvn exec` style test. Are we sure we want that?

Reply:
Maybe not. I just feel a bit uncomfortable with having no tests at all. But we can wait for the first fail and I will "I told you so" you ;).

More seriously, if it's not convenient, we can probably let it slide for now.

Reply:
Good question. 

AFAIK, the resources are generated after the test phase which makes testing a little bit weird.

I assume its not the first time this issue arises. How do you deal with it?

Reply:
I am uneasy with not having tests as well, but on the other hand those `mvn exec` style tests have caused a lot of headaches. 
I propose we skip for now and when I receive the "I told you so" sign I can add them :P

Reply:
So now there are tests :)